src/style.css

```
@@ -1,3 +1,73 @@
-body {
+*,
+*::after,
+*::before {
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo {
+  display: flex;
+  align-items: center;
+  height: 60px;
+}
+
+.nav-ul {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 60px;
+  padding: 0;
+  margin: 0;
+  list-style-type: none;
+}
+
+.li-nav {
+  position: relative;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  text-transform: uppercase;
+  margin-left: 20px;
+}
+
+.li-nav:first-of-type {
+  margin-left: 0;
+}
+
+.li-nav:hover {
+  color: #00acdc;
+  cursor: pointer;
+}
+
+.element {
+  display: flex;
+  align-items: center;
+  position: relative;
+  font-family: "Roboto", sans-serif;
+  height: 60px;
+  color: inherit;
+  text-decoration: none;
+}
+
+a.is-active {
+  color: #00acdc;
+}
+
+a.is-active::after {
```

```suggestion
.is-active {
  color: #00acdc;
}

.is-active::after {
```

We shouldn't attach the `<a/>` tag with `is-active` class name, maybe we will want to use this class on `div` also 🤔